src/main/java/org/example/ngevaticketmanagerspring/user/UserController.java

```diff
@@ -0,0 +1,72 @@
+package org.example.ngevaticketmanagerspring.user;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/api/user", produces = MediaType.APPLICATION_JSON_VALUE)
+public class UserController {
+
+    private final UserService service;
+
+    @Autowired
+    public UserController(final UserService service) {
+        this.service = service;
+    }
+
+    @PostMapping
+    public ResponseEntity<User> createUser(@RequestBody User user) {
```

Comment:
```suggestion
    public ResponseEntity<User> createUser(@Valid @RequestBody User user) {
```

Ich habe hier noch @Valid, um das Objekt direkt bei Eingang zu validieren.